build.sh

```diff
@@ -18,10 +18,15 @@
 
 SCRIPT_PATH="$(cd "$(dirname "$0")" && pwd -P)"
 
+# Build assets
+${SCRIPT_PATH}/asset.sh
```

Comment:
Okay, I see you've added the `asset.sh` invocation here - but are we sure that all of the following are available on PHP containers on Travis?

- npm
- gulp
- perl

I know python and pip are, but I had to test that first to be sure. Can you check and verify, please?